Tools-Override/resources.targets

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@alexperovich I can't remember much about this, but <DisableOutOfProcTaskHost>true</> may be part of the solution.